database/migrations/Concerns/CanModifyPermissions.php

```diff
@@ -90,7 +90,7 @@ public function createPermissions(array $names, string $guardName): void
     /**
      * @param array<string> $names
      */
-    public function deletePermissions(array $names, string $guardName): void
+    public function deletePermissions(array $names, string|array $guardName): void
```

Comment:
This should not take an array, an array being passed in would not work. What was the purpose for this change?